src/tests/unit/tests/electron/views/results/components/reflow-command-bar.test.tsx

```diff
@@ -72,7 +72,7 @@ describe('ReflowCommandBar', () => {
                 allowsExportReport: true,
                 startOverButtonSettings: _ => {
                     return {
-                        onClick: () => {},
+                        onClick: _ => {},
```

Comment:
GitHub won't let me leave a comment far away from a change (grumble), but it'd be nice if the interaction test down on L134 could verify that the expected event stub gets passed through

Reply:
Updated!